src/improvements/tasks/MultisigTask.sol

```diff
@@ -630,11 +717,19 @@ abstract contract MultisigTask is Test, Script {
     function printNestedHashToApprove() public view {
         bytes memory callData = generateApproveMulticallData();
 
+        // this branch means the function `signFromChildMultisig` is being called
```

Comment:
I think we will have to revisit the general architecture of MultisigTask.sol. This comment is good in the meantime, but if we move to a more functional style and away from having to keep track of global state variable changes, I think we will gain a lot more readability.

Reply:
Tracking in https://github.com/ethereum-optimism/superchain-ops/issues/636, so resolving this